WIP: Sanity test to validate galaxy.yml metadata file for the required keys (#75335)
ansible/ansible: docs/docsite/rst/dev_guide/testing/sanity/galaxy-metadata.rst

galaxy-metadata
===============

* Validates the required keys (namespace, name, version, readme, authors) for the collection galaxy.yml metadata file.

> * If the required keys aren’t present or if the required and other keys contain ``null`` value in the galaxy.yml file then this sanity test will throw an error.

Reviewer: ```suggestion
* If the required keys are not present or if the required and other keys contain ``null`` value in the ``galaxy.yml`` file, this sanity test will throw an error.
```
